.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -49,6 +49,10 @@ CONDITION: Cherry-picking into LTS. Otherwise, delete.
 
 <_YOUR_REASONS_>
 
+## List the steps to manually verify the changes in this cherry-pick
```

Comment:
Should we include a link to an example of the steps?

Reply:
Do you have a good example to suggest? I've currently stuck to the convention in the template by including a `<TEST_STEPS>` placeholder for the CP author to fill out.

Reply:
Merging for now, will incrementally improve this as developers offer feedback.